syntax = "proto3";

message EchoRequest {

        string request = 1;
}

message EchoResponse {
        string response = 1;
}

service EchoService {
        rpc echo(EchoRequest) returns (EchoResponse);
}
